dotnet/nuget/nuget-package.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <!-- Central version prefix - applies to all nuget packages. -->
-    <Version>0.26</Version>
+    <Version>1.0.0-rc1</Version>
```

Comment:
We are going to go with `1.0.0-beta1` for this release and switch to `1.0.0-rc1`, later likely when we remove all of the obsolete stuff

Reply:
The version has been updated to 1.0.0-beta1